### PR TITLE
Added loading parameters from info.plist

### PR DESCRIPTION
**Pull request by kekkokk**
Added a method to load configuration from info.plist to easily tune user preferences.

### CODE REVIEW

**Comment by leo-labs** [comment]
Thanks for the pull request, I will have a look at it in the next days!

File: VoodooSMBus/Info.plist [hunk 1]
```diff
@@ -37,6 +37,15 @@
 		</dict>
 		<key>ELANTouchpadDriver</key>
 		<dict>
+			<key>Preferences</key>
+			<dict>
+				<key>afterTypingCooldownMs</key>
+				<integer>500</integer>
+				<key>ignoreVoodooDisableTrackpad</key>
```
Comment (leo-labs) [inline VoodooSMBus/Info.plist]: Is there a reason why one would want to do that?

File: VoodooSMBus/Info.plist [hunk 2]
```diff
@@ -37,6 +37,15 @@
...
+				<false/>
+				<key>palmDetection</key>
```
Comment (leo-labs) [inline VoodooSMBus/Info.plist]: Could you rename this to `disableWhileTyping` ? Palm detection would be actually detecting if its a palm based on the touch ellipses and I want to add that later.

File: VoodooSMBus/ELANTouchpadDriver.cpp [hunk 3]
```diff
@@ -29,10 +29,69 @@
 
 OSDefineMetaClassAndStructors(ELANTouchpadDriver, VoodooSMBusSlaveDeviceDriver);
 
+void ELANTouchpadDriver::loadConfiguration() {
```
Comment (leo-labs) [inline VoodooSMBus/ELANTouchpadDriver.cpp]: I think this method is overly complicated. I would prefer to have a `loadBoolConfiguration(char* name)` and a `loadUInt64Configuration(char* name)` inside `helper` so we are able to reuse them inside other classes. Especially we should not need to iterate over the whole configuration. Do you want to make this change or should I?

Comment (kekkokk) [reply VoodooSMBus/ELANTouchpadDriver.cpp]: I don't have time so please, feel free to change whatever you want

**Comment by leo-labs** [comment]
I am closing this PR because github actions do not seem to work yet for PRs from forks where they are not enabled yet.

**Comment by leo-labs** [comment]
Reopened in #15

**Pull request closed without merging**